backend/lucene/src/main/java/org/hibernate/search/backend/lucene/ValueSortField.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.backend.lucene;
+
+import org.apache.lucene.search.FieldComparatorSource;
+import org.apache.lucene.search.Query;
+import org.apache.lucene.search.SortField;
+import org.apache.lucene.search.join.ScoreMode;
+
+/**
+ *
+ * @author Waldemar Kłaczyński
+ */
+public class ValueSortField extends SortField {
```

Comment:
This is unrelated to the ticket and we can expose filtered sorts without this.

I will not merge this as part of this PR, but you can find the corresponding changeset [here](https://github.com/yrodiere/hibernate-search/commit/e0f771fa20f372632a03ab48ed2bb0420a241f25) should you ever need it.